jenkins/client.sls

```diff
@@ -0,0 +1,40 @@
+{% from "jenkins/map.jinja" import client with context %}
+{%- if client.enabled %}
+
+jenkins_client_install:
+  pkg.installed:
+  - names: {{ client.pkgs }}
+
+jenkins_client_dirs:
+  file.directory:
+  - name: /srv/salt/env/dev/_jenkins_jobs
```

Comment:
Needs to be in salt file root, soooo only thing ugly here is it's not being parametrised :wink: 
I'll into it, and ```/var/cache/salt``` might be even better